tests/lora/utils.py

```diff
@@ -2239,11 +2251,15 @@ def initialize_pipeline(storage_dtype=None, compute_dtype=torch.float32):
         pipe_fp32 = initialize_pipeline(storage_dtype=None)
         pipe_fp32(**inputs, generator=torch.manual_seed(0))[0]
 
-        pipe_float8_e4m3_fp32 = initialize_pipeline(storage_dtype=torch.float8_e4m3fn, compute_dtype=torch.float32)
-        pipe_float8_e4m3_fp32(**inputs, generator=torch.manual_seed(0))[0]
+        # MPS doesn't support float8 yet.
+        if torch_device not in {"mps"}:
+            pipe_float8_e4m3_fp32 = initialize_pipeline(storage_dtype=torch.float8_e4m3fn, compute_dtype=torch.float32)
+            pipe_float8_e4m3_fp32(**inputs, generator=torch.manual_seed(0))[0]
 
-        pipe_float8_e4m3_bf16 = initialize_pipeline(storage_dtype=torch.float8_e4m3fn, compute_dtype=torch.bfloat16)
-        pipe_float8_e4m3_bf16(**inputs, generator=torch.manual_seed(0))[0]
+            pipe_float8_e4m3_bf16 = initialize_pipeline(
+                storage_dtype=torch.float8_e4m3fn, compute_dtype=torch.bfloat16
+            )
+            pipe_float8_e4m3_bf16(**inputs, generator=torch.manual_seed(0))[0]
```

Comment:
This is unrelated. So, please open a new PR for this.

Reply:
Done in https://github.com/huggingface/diffusers/pull/10216/commits/e97a83ea7ed53b1ed27834bcbe390bbbc6ab28b3.